teambit.bvm/commands/list/list.ts

```diff
@@ -1,43 +1,47 @@
-import type {CommandModule, Argv} from 'yargs';
-import {listRemote, listLocal} from '@teambit/bvm.list';
+import type { CommandModule, Argv } from 'yargs';
+import { listRemote, listLocal } from '@teambit/bvm.list';
 
 export class ListCmd implements CommandModule {
   aliases = ['l', 'list'];
   describe = 'list installed bit versions';
-  command = [
-    'list'
-  ];
+  command = ['list'];
   builder(yargs: Argv) {
     yargs
-    .option({
-      remote: {
-        alias: ['r'],
-        describe: 'show versions exist in the remote server',
-        default: false,
-        type: 'boolean'
-      }
-    })
-    .example('$0 list', 'show all installed versions')
-    .example('$0 list --remote', 'show all versions available for install')
+      .option({
+        remote: {
+          alias: ['r'],
+          describe: 'show versions exist in the remote server',
+          default: false,
+          type: 'boolean',
+        },
+        'release-type': {
+          alias: ['t'],
+          describe: 'show only the stable versions',
+          default: 'stable',
+          type: 'string',
```

Comment:
make it choices  - https://github.com/yargs/yargs/blob/master/docs/api.md#choiceskey-choices

Reply:
fixed